docs/hopsworksai/sso/oauth.md

```diff
@@ -0,0 +1,191 @@
+# Configure your hopsworks cluster to use OAuth2 for user management.
+Once you have created a Hopsworks cluster you can configure it to use OAuth2 for its access control.
+We will go through a step-by-step description of the configuration process.
+To illustrate our explanation We will use Azure Active Directory as the identity provider, but the same can be done with any resource provider supporting OAuth2.
+
+## Step 1: Configure your identity provider.
+To use OAuth2 in hopsworks you first need to create and configure an OAuth client in your resource provider. We will take the example of Azure AD for the remaining of this documentation, but equivalent steps can be taken on other identity providers.
+
+Navigate to the [Microsoft Azure Portal](https://portal.azure.com) and authenticate. Navigate to [Azure Active Directory](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Overview). Click on [App Registrations](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps). Click on *New Registration*.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/sso/create_application.png">
```

Comment:
should be ../../assets. When deployed it will have version in the path

Reply:
No, this is correct because I am in docs/hopsworksai/sso/ which is three folders deep while others are only 2 folders deep. I tested it.